src/script/conversation/ConversationRepository.ts

```diff
@@ -2714,11 +2719,9 @@ export class ConversationRepository {
         eventJson.from = this.userState.self().id;
       }
 
-      if (conversationEntity.protocol === ConversationProtocol.MLS) {
+      if (isMLSCapableConversation(conversationEntity)) {
```

Comment:
Whenever we leave (or get removed) from mixed / mls conversation we have to wipe its MLS group.